### PR TITLE
update README with new features

### PR DESCRIPTION
**Pull request by sonicaghi**
my last marketing commit!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ through plugins.
   Kubernetes.
 - **Dynamic Load Balancing**: Load balance traffic across multiple backend
   services.
+- **Hash-based Load Balancing**: Load balance consisten hashing/sticky sessions.
+- **Circuit-Breaking**: Smarter trackling of unhealthy upstream instances.
```
Comment (coopr) [inline README.md]: "trackling" -> "tracking"

File: README.md [hunk 2]
```diff
@@ -43,6 +43,9 @@ through plugins.
   Kubernetes.
 - **Dynamic Load Balancing**: Load balance traffic across multiple backend
   services.
+- **Hash-based Load Balancing**: Load balance consisten hashing/sticky sessions.
```
Comment (coopr) [inline README.md]: (Looks like you might want to enable spell checking on your editor :)

"consisten" -> "consistent"

File: README.md (hunk 1 above)
Comment (coopr) [reply README.md]: Also, "Circuit Breaking" isn't really a term we use - "Circuit Breaker" is more appropriate, and better SEO

File: README.md [hunk 3]
```diff
@@ -56,6 +59,7 @@ through plugins.
 - **SSL**: Setup a Specific SSL Certificate for an underlying service or API.
 - **Monitoring**: Live monitoring provides key load and performance server
   metrics.
+- **Forward Proxy**: Allows Kong to connect to intermadiary transparent HTTP proxies.
```
Comment (coopr) [inline README.md]: "intermadiary" -> "intermediary"

File: README.md [hunk 4]
```diff
@@ -43,6 +43,9 @@ through plugins.
   Kubernetes.
 - **Dynamic Load Balancing**: Load balance traffic across multiple backend
   services.
+- **Hash-based Load Balancing**: Load balance consisten hashing/sticky sessions.
+- **Circuit-Breaking**: Smarter trackling of unhealthy upstream instances.
+- **Health Checks:** Passive and Active monitoring of your services.
```
Comment (jbergstroem) [inline README.md]: Any reason for "Active" to be capitalized here?

Comment (sonicaghi) [reply README.md]: @jbergstroem will fix thanks

**Pull request merged**